docs/howto/use-alternative-secrets-backend.rst

```diff
@@ -351,6 +351,7 @@ GCP Secrets Manager Backend
 
 To enable GCP Secrets Manager to retrieve connection/variables, specify :py:class:`~airflow.providers.google.cloud.secrets.secrets_manager.CloudSecretsManagerBackend`
 as the ``backend`` in  ``[secrets]`` section of ``airflow.cfg``.
+Displaying secrets from the GCP Secrets Manager backend is not yet supported by the Web UI. 
```

Comment:
This sentence is true for all backends

Reply:
I shall move it then

Reply:
Wait, what do you mean by secrets here though, it is consistent with how Connections defined by Environment Variables were not viewable even before we introduced the concepts of Secrets Backends.

Reply:
Yes, it's consistend. From what I understand it's the same underlying principle. Is that already mentioned somewhere in the doc?

Reply:
For Variables, we it at https://airflow.readthedocs.io/en/latest/concepts.html#storing-variables-in-environment-variables

>Variables set using Environment Variables would not appear in the Airflow UI but you will be able to use it in your DAG file.

For Connection, we have details on how to set it using Env Var but we don't state it that it won't be visible in the UI. 

May be add something as a Sphinx "Note" / "Info" / "Warning" to say that the UI would only show Connections & Variables stored in Metadata DB

Reply:
I added your suggestion to the doc.

Reply:
I also cleaned up the commit message